examples/onnxruntime/quantization/image-classification/run_image_classification.py

```diff
@@ -32,6 +32,7 @@
 from datasets import load_dataset, load_metric
 from torchvision.transforms import CenterCrop, Compose, Normalize, Resize, ToTensor
 from transformers import AutoFeatureExtractor, EvalPrediction, HfArgumentParser, TrainingArguments
+from transformers.onnx import FeaturesManager
```

Comment:
Also before merging, I think we should update the ort [notebook](https://github.com/huggingface/notebooks/blob/main/examples/text_classification_quantization_ort.ipynb)

Reply:
Lets update the Notebooks once the PR is merged and before the release since we are not installing from main

Reply:
We plan a release for the `ORTQuantizer` / `ORTOtptimizer` refactorization, so the sooner the better to be able to freely to the release in my opinion

Reply:
> We plan a release for the ORTQuantizer / ORTOtptimizer refactorization, so the sooner the better to be able to freely to the release in my opinion

When is the release planned? As said lets get it in then quickly update the notebook